project-base/config/services_test.yaml

```diff
@@ -75,3 +75,5 @@ services:
     Shopsys\FrameworkBundle\Model\Pricing\Rounding: ~
 
     Shopsys\FrontendApiBundle\Model\Token\TokenFacade: ~
+
+    Shopsys\FrameworkBundle\Component\Setting\Setting: ~
```

Comment:
This is no longer necessary as Setting class is mocked in test

Reply:
fixed